### PR TITLE
Updated google-auth to latest

### PR DESCRIPTION
**Pull request by agraubert**
Fair warning: I only tested this briefly, but it seems like it's working fine

### CODE REVIEW

**Comment by dheiman** [comment]
Last I checked, if google-auth is set to latest, it would emit a deprecation warning with every API call due to the use of application-default credentials. That's why I froze it at the last version that didn't do so. Is there a pressing reason to update?

**Comment by agraubert** [comment]
It's causing dependency conflicts; specifically, I can't update the version used by lapdog because fiss has it pinned here.
Recent versions of pip have, annoyingly, started running hard dependency checks when launching console scripts, so if I were to update to `google-auth==1.6.3` and then run lapdog, I encounter an error because I'm conflicting with fiss' requirements.

Added a new commit with a filter to silence the warning

File: setup.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def get_outputs(self):
     },
     test_suite = 'nose.collector',
     install_requires = [
-        'google-auth==1.4.2',
+        'google-auth==1.6.3',
```
Comment (dheiman) [inline setup.py]: won't you run into this issue again if you pin the version here?

**Comment by dheiman** [comment]
Also, could you make this pull request against the rc-0.16.22 branch? That way master will stay relatively clean.

File: setup.py (hunk 1 above)
Comment (agraubert) [reply setup.py]: Yes, but I wasn't sure if you waned to keep the pin. I can just change it to `>=`

**Pull request merged**